app/models/category.rb

```diff
@@ -1,3 +1,8 @@
 class Category < ApplicationRecord
-  has_many :tests
-end
\ No newline at end of file
+  has_many :tests,
+            dependent: :destroy
```

Comment:
```suggestion
  has_many :tests, dependent: :destroy
```
я бы сделал в одну строку, обычно бьем на несколько, когда длинная строка получается, а тут вроде норм